argo/providers/stub/Dockerfile

```diff
@@ -1,9 +1,10 @@
 FROM alpine:3.20.1
 
 WORKDIR /
+
 COPY inject.sh inject.sh
 COPY compile.sh compile.sh
 COPY bin/provider .
-USER 65532:65532
```

Comment:
originally used an ID that didn't exist, it now uses the consistent `nobody` user